contextily/tile.py

```diff
@@ -113,6 +117,17 @@ def bounds2raster(
         [Optional. Default: 2]
         total number of rejected requests allowed before contextily
         will stop trying to fetch more tiles from a rate-limited API.
+    n_connections: int
+        [Optional. Default: 1]
+        number of connections for downloading tiles in parallel.
+    max_connections: int
+        [Optional. Default: 32]
+        Maximum number of connections for downloading tiles in parallel. Be careful not to overload the tile server if
+        you increase this value.
+    disable_cache: bool
+        [Optional. Default: False]
+        If True, caching of the downloaded tiles will be disabled. This can be useful in resource constrained
+        environments, especially when using n_connections > 1.
```

Comment:
```suggestion
        If True, caching of the downloaded tiles will be disabled. This can be useful in resource constrained
        environments, especially when using n_connections > 1 or when a tile provider's terms of use don't
        allow caching.
```